code/lib/cli/src/upgrade.ts

```diff
@@ -138,7 +145,13 @@ export const doUpgrade = async ({
     throw new UpgradeStorybookToSameVersionError({ beforeVersion });
   }
 
-  const latestVersion = await packageManager.latestVersion('@storybook/cli');
+  const [latestVersion, packageJson, storybookVersion] = await Promise.all([
+    //
+    packageManager.latestVersion('@storybook/cli'),
+    packageManager.retrievePackageJson(),
+    getCoercedStorybookVersion(packageManager),
+  ]);
```

Comment:
Testcase: Upgrade Storybook 7 to 8 in a Vue 2 project.

Issue:
Storybook aborts the upgrade, because it cannot determine the Storybook version so that the automigrations will be skipped.

Error message: info [Storybook automigrate] ❌ Unable to determine Storybook version so that the automigrations will be skipped.

Reason:
`getCoercedStorybookVersion` returns the version of the used renderer package. The `rendererPackage` defined in `code/lib/core-common/src/utils/get-storybook-info.ts` doesn't mention/list `@storybook/vue`, since we removed it in Storybook 8. For the migration process this is unfortunate.

We have some possibilities here:

1) We add `@storybook/vue` back to the `rendererPackages`. I don't like this, because this can have some side effects. Where is `rendererPackages` used as well? Does it have any negative effects, if we list renderers/frameworks, which aren't supported in the current version anymore? This can have so many bad effects.
2) We use `getInstalledStorybookVersion` instead. Downside is, that in Storybook 6, neither `storybook` nor `@storybook/cli` are required being installed as far as I know. Does a Storybook package exist throughout version 6 to 8, which is always installed? Something like `@storybook/core-common`?